src/native/minipal/utf8.c

```diff
@@ -0,0 +1,2172 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <minipal/utf8.h>
+
+#include <errno.h>
+#include <limits.h>
+#include <string.h>
+#include <assert.h>
+
+#define FASTLOOP
```

Comment:
Is it necessary to have this unconditional define?

Reply:
It can be cleaned up.